### PR TITLE
CASSANDRA-9749 cleanup

### PR DESCRIPTION
**Pull request by mambocab**
This cleans up #375 for merging. @bdeggleston can you review my changes please? In particular, I'd like confirmation that my new assertion logic is correct. Thanks again for these tests!

@ptnapoleon can you review as well?

### CODE REVIEW

**Comment by bdeggleston** [comment]
Looks good to me

File: commit_log_test.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+        # check that ks.tbl hasn't been flushed
+        path = node.get_path()
+        ks_dir = path + '/data/ks'
+        db_dir = os.listdir(ks_dir)[0]
+        sstables = len([f for f in os.listdir(os.path.join(ks_dir, db_dir)) if f.endswith('.db')])
+        self.assertEqual(sstables, 0)
+
+        # modify the commit log crc values
+        cl_dir = path + '/commitlogs'
```
Comment (ptnapoleon) [inline commit_log_test.py]: :(

File: commit_log_test.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        # with the commitlog_sync_period_in_ms set to 1000,
+        # this sleep guarantees that the commitlog data is
+        # actually flushed to disk before we kill -9 it
+        time.sleep(1)
+
+        node.stop(gently=False)
+
+        # check that ks.tbl hasn't been flushed
+        path = node.get_path()
+        ks_dir = path + '/data/ks'
```
Comment (ptnapoleon) [inline commit_log_test.py]: :(

**Comment by ptnapoleon** [comment]
Always use `os.path.join` to merge paths, rather than doing it yourself.

**Comment by ptnapoleon** [comment]
Other than that, seems fine.

**Comment by mambocab** [comment]
grepped for `/` and `+` and subbed in more `os.path.join`s. Can I get another once-over, please?

**Comment by ptnapoleon** [comment]
+1

**Comment by mambocab** [comment]
Thank you. Merging.

**Pull request merged**